properties/material.py

```diff
@@ -24,5 +24,6 @@ def update_preview(self, context):
 
 
 class LuxCoreMaterialProps(PropertyGroup):
+    auto_mat_selection = bpy.props.BoolProperty(name="Automatic Material Selection", default=True)
```

Comment:
It's not really a "material selection". This was a bit of a typo I did in the review.
I think something like "Auto Viewport Color" would be a better name.